### PR TITLE
catt: 0.11.0 -> 0.12.0

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update to latest upstream release 0.12.0

`catt` now support later `PyChromecast` releases.

Change log: https://github.com/skorokithakis/catt/releases/tag/v0.12.0

Update `PyChromecast` in the process.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 114026` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages built:</summary>
  <ul>
    <li>catt</li>
    <li>gnomecast</li>
    <li>mkchromecast</li>
    <li>python37Packages.PyChromecast</li>
    <li>python38Packages.PyChromecast</li>
    <li>python39Packages.PyChromecast</li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 114026` at 3db167c6 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/4dce3e7c908fa75e19907041cd3f4e77">catt</a></li></li>
    <li><a href="https://gist.github.com/989d6f868fdf8732f167592d50e2f704">gnomecast</a></li></li>
    <li><a href="https://gist.github.com/f96358ab5b1b4d49d3b2a4302776f558">mkchromecast</a></li></li>
    <li><a href="https://gist.github.com/c2ece50cac2fc6a3fe79f1bc0ec9a9a8">python37Packages.PyChromecast</a></li></li>
    <li><a href="https://gist.github.com/7d8a6a29399696f98c0d43bf18bd1988">python38Packages.PyChromecast</a></li></li>
    <li><a href="https://gist.github.com/c72a224f16206e01d12fa744a5c57673">python39Packages.PyChromecast</a></li></li>
  </ul>
</details>

File: pkgs/applications/video/catt/default.nix [hunk 1]
```diff
@@ -1,32 +1,26 @@
...
-      PyChromecast = super.PyChromecast.overridePythonAttrs (oldAttrs: rec {
-        version = "6.0.0";
-        src = oldAttrs.src.override {
-          inherit version;
-          sha256 = "05f8r3b2pdqbl76hwi5sv2xdi1r7g9lgm69x8ja5g22mn7ysmghm";
-        };
-      });
-    };
-  };
+with python3.pkgs;
```
Comment (SuperSandro2000) [inline pkgs/applications/video/catt/default.nix:3]: This should also be removed.

Comment (Mic92) [reply pkgs/applications/video/catt/default.nix:3]: When you do python3Packages.callPackages in all-packages.nix for this file, than you can just put all python packages into the arguments above.

**Pull request merged**